src/components/e-picture.vue

```diff
@@ -278,6 +278,8 @@
 </script>
 
 <style lang="scss">
```

Comment:
Replace `padding-top: calc(100% / var(--aspect-ratio));` by `padding-top: calc(#{math.div(100%, var(--aspect-ratio))});`

Reply:
@janwidmer I guess inside the `calc()` function a division with "/" is still ok not? Because `calc()` is a CSS function and not a SASS Function.
=> In the documentation of the `calc()` function is nothing mentioned that you should prevent using "/" for divisions. https://developer.mozilla.org/de/docs/Web/CSS/calc()

Reply:
that's true.. :-)